src/components/VerificationCode/__tests__/__snapshots__/index.js.snap

```diff
@@ -244,7 +244,7 @@ exports[`VerificationCode should handle and replace non number with "" when type
       pattern="[0-9]*"
       placeholder=""
       type="tel"
-      value="3"
+      value=""
```

Comment:
Intentional ?

Reply:
Nope that's strange I can't reproduce out of tests 🤔